tests/lib/prepare-restore.sh

```diff
@@ -675,6 +687,42 @@ restore_suite_each() {
         "$TESTSTOOLS"/cleanup-state pre-invariant
     fi
     tests.invariant check
+
+    if os.query is-classic; then
```

Comment:
Hm perhaps this should run earlier, before tests.cleanup restore and tests.backup restore, otherwise it may pick up file changes that were part of restore actually? Perhaps that's why it picked up the original core revision being manipulated.

Reply:
done